mean_image_handler.py

```diff
@@ -203,6 +204,35 @@ def postprocess(self, inference_output):
         postprocess_output = inference_output
         return postprocess_output
 
+    def generate_animation(self, encodings, FPS=25, duration_per_image=1):
+        encodings = [encoding.squeeze() for encoding in encodings]
+        animation_frames = []
+
+        print('Generating morphing animation')
+        for i, latent in enumerate(interpolate(
```

Comment:
Is it efficient to do this loop here? Would it be better to add this to the interpolate function?